src/main/java/core/basesyntax/FigureType.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public enum FigureType {
+    SQUARE, RECTANGLE, RIGHT_TRIANGLE, CIRCLE, ISOSCELES_TRAPEZOID
+}
```

Comment:
it's redundant, you don't use those values, just generated based off integers

Reply:
not fixed